src/transformers/models/albert/modeling_tf_albert.py

```diff
@@ -931,8 +955,28 @@ def __init__(self, config, *inputs, **kwargs):
         self.albert = TFAlbertMainLayer(config, name="albert")
         self.predictions = TFAlbertMLMHead(config, self.albert.embeddings, name="predictions")
 
-    def get_output_embeddings(self):
```

Comment:
I'd really like to keep this API - completely removing the function name is very breaking in my opinion. This is not really a private method.

Reply:
What does-it breaks more precisely?

Reply:
I mean I would think that people made use of this method for example if they wrap their language model in a custom class or if they built their own generation method